src/ui/components/DataTable/CustomizeColumns.tsx

```diff
@@ -1,134 +1,148 @@
-import type { Col } from ".";
 import { useState } from "react";
-import { Modal } from "react-bootstrap";
-import { SortableContainer, SortableElement } from "react-sortable-hoc";
-import classNames from "classnames";
+import {
+	Col as FlexCol,
```

Comment:
Rather than using Col/Row/ListGroup from react-bootstrap, just use the Bootstrap CSS directly. That's what I do elsewhere for these features. I only use react-bootstrap if there is some React-specific interactivity I need.

Reply:
made change